### PR TITLE
[FEATURE] Scheduling: Date&Time component

### PR DESCRIPTION
**Pull request by josipbagaric**

### CODE REVIEW

File: common/components/SendButtonFactory/OfflineBroadcast.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ export const OfflineBroadcast = connect((state: AppState) => ({ offline: getOffl
 const BroadCast: React.SFC<{}> = () => (
   <p>
     To broadcast this transaction, paste the above into the{' '}
-    <Link to="pushTx">Broadcast Transaction tab</Link> or{' '}
+    <Link to="pushTx">Schedule Transaction tab</Link> or{' '}
```
Comment (e00dan) [inline common/components/SendButtonFactory/OfflineBroadcast.tsx]: Hey, are you sure about this change? Remember you're changing original component not a cloned/copied and renamed one.

Comment (josipbagaric) [reply common/components/SendButtonFactory/OfflineBroadcast.tsx]: Yeah this is an invalid change. Changed that when I was making the screenshots. Will undo.

**Comment by josipbagaric** [comment]
Ready to be merged. Please review @Kuzirashi @kosecki123

File: common/selectors/transaction/transaction.ts [hunk 2]
```diff
@@ -62,15 +74,18 @@ const getSchedulingTransaction = (state: AppState): IGetTransaction => {
...
   const gasPrice = getGasPrice(state);
   const timeBounty = getTimeBounty(state);
   const windowStartValid = isWindowStartValid(transactionFields, getLatestBlock(state));
+  const scheduleTimestampValid = isScheduleTimestampValid(transactionFields);
 
   const isFullTransaction =
     isFullTx(state, transactionFields, currentTo, currentValue, dataExists, validGasCost, unit) &&
-    windowStartValid;
+    windowStartValid &&
+    scheduleTimestampValid;
```
Comment (e00dan) [inline common/selectors/transaction/transaction.ts:88]: :clap: Perfect :)

File: common/selectors/transaction/scheduleTimestamp.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+const isValidCurrentScheduleTimestamp = (state: AppState) => {
+  const currentScheduleTimestamp = getScheduleTimestamp(state);
+  const raw = currentScheduleTimestamp.raw;
+  const value = currentScheduleTimestamp.value;
+
+  const fiveMinFromNow = moment()
+    .add(5, 'm')
+    .toDate();
+
+  if (!value || value <= fiveMinFromNow || !isCorrectFormat(raw)) {
```
Comment (e00dan) [inline common/selectors/transaction/scheduleTimestamp.ts]: This could be one line:

`return value && value > fiveMinFromNow && isCorrectFormat(raw);`

**Review by e00dan — APPROVED** [review]

**Review by e00dan — APPROVED** [review]

**Pull request merged**